core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1674,7 +1674,11 @@ private[spark] object Utils extends Logging {
 
   /**
    * NaN-safe version of `java.lang.Double.compare()` which allows NaN values to be compared
-   * according to semantics where NaN == NaN and NaN &gt; any non-NaN double.
+   * according to semantics where
+   *
+   * {{{
+   * NaN == NaN and NaN > any non-NaN double
+   * }}}.
```

Comment:
- Java
  <img width="816" alt="2016-11-27 3 21 11" src="https://cloud.githubusercontent.com/assets/6477701/20642378/9ea18b9c-b450-11e6-80df-4bf6d04d20aa.png">


Scaladoc not found